README.md

```diff
@@ -77,6 +79,12 @@ Runtime has the following third-party library dependencies:
 	1. Install `doxygen`: `sudo apt-get -y install doxygen`
 	2. Repeat steps 5.i and 5.ii from above in the `protobuf-c` directory, then do `make html`
 	3. Then navigate to that directory in Explorer/Finder and open the `html/index.html` file by double clicking on it
+	
+#### Installing `ncurses`
+
+The command below should install on Raspbian / Debian machines:
+
+1. `sudo apt-get install libncurses5-dev libncursesw5-dev`
 
```

Comment:
I also added a similar section on my shm UI branch! Not sure what steps to take to resolve the potential conflict

Reply:
Ah ok if that's the case I'll revert these changes and the changes from your branch should be added--makes more sense~~
I thought that yours would get merged in soon and I didn't remember seeing it here, which is why I added this, my bad!